src/config/dynamic-theme-fixes.config

```diff
@@ -3365,6 +3365,10 @@ INVERT
 img[src$="google_gsuite"]
 div[role="menuitem"] svg
 div[role="dialog"] div[role="button"][tabindex="0"]
+img[src="https://drive-thirdparty.googleusercontent.com/128/type/audio/mp3"]
+img[src="https://drive-thirdparty.googleusercontent.com/32/type/audio/mp3"]
+div[role="menu"] > div[role="menuitem"] > div > div > div
```

Comment:
These 2 selectors will cause a lot of other conversion other than menu buttons, they should be covered by Line 3377.

Reply:
Should I remove both of them in favour of 3377 then? Because it doesn't seem to work for me:
![image](https://user-images.githubusercontent.com/71190696/118694971-485cf980-b7d2-11eb-9eba-32d62307db2f.png)
I erased the lines I added in mine and updated it.

Reply:
Hmm that shouldn't happens and that header shouldn't look inverted.

Reply:
You can remove the line: #5930
```suggestion

````